README.md

```diff
@@ -46,6 +46,17 @@ powershell
 Set-ExecutionPolicy unrestricted
 ```
 
+#### If build with PowerShell does not work
+If you get errors like "Error: Java heap space" or "dx tool failed", you probably have to allocate more Java memory:
+- open the jvm.config file inside your Flex Air SDK directory and set at least 512 Mb
+```
+java.args=-Xmx512m
+```
+- create the _JAVA_OPTIONS system environment variable 
```

Comment:
Is this required? Simply setting the jvm.args should handle this. If this setting is handling the heap space, what is the purpose of the config file and the reverse also?

Reply:
Yes, for me it's needed, I found solution here: http://www.flashdevelop.org/community/viewtopic.php?f=6&t=12995